server.py

```diff
@@ -169,23 +211,71 @@ async def state():
                 }
     return  { "state": State.INTERLUDE }
 
+@app.post("/play/file")
+async def play_file(file_path: str="cache", title: str="", thumbnail: str=""):
+    # Check if video is already playing
+    if State.PLAYING in process_dict:
+        raise HTTPException(status_code=409, detail="Please wait for the current video to end, then make the request")
+    
+    # Store the current playing video information if any
+    if title != "" and thumbnail != "":
+        current_video_dict["title"] = title
+        current_video_dict["thumbnail"] = thumbnail
+
+    # Check if interlude is playing
+    if State.INTERLUDE in process_dict:
+        # Stop interlude
+        stop_video_by_type(State.INTERLUDE)
```

Comment:
```suggestion
```suggestion
    # Store the current playing video information if any
    if title != "" and thumbnail != "":
        current_video_dict["title"] = title
        current_video_dict["thumbnail"] = thumbnail

    for video_type in State:
        # Stop the video playing subprocess
        stop_video_by_type(video_type)
```
```